module/fastgltf.cppm

```diff
@@ -0,0 +1,162 @@
+module;
+
+#include <cassert>
+#ifndef FASTGLTF_USE_STD_MODULE
+#include <fastgltf/core.hpp>
+#endif
+
+export module fastgltf;
+
+#ifdef FASTGLTF_USE_STD_MODULE
+import std;
+import std.compat;
+
+extern "C++" {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Winclude-angled-in-module-purview"
+#include <fastgltf/core.hpp>
+#pragma clang diagnostic pop
+}
+#endif
+
+export namespace fastgltf {
+namespace math {
+    using math::vec;
+    using math::mat;
+    using math::sum;
+    using math::dot;
+    using math::cross;
+    using math::length;
+    using math::normalize;
+    using math::s8vec2;
+    using math::s8vec3;
+    using math::s8vec4;
+    using math::u8vec2;
+    using math::u8vec3;
+    using math::u8vec4;
+    using math::s16vec2;
+    using math::s16vec3;
+    using math::s16vec4;
+    using math::u16vec2;
+    using math::u16vec3;
+    using math::u16vec4;
+    using math::s32vec2;
+    using math::s32vec3;
+    using math::s32vec4;
+    using math::u32vec2;
+    using math::u32vec3;
+    using math::u32vec4;
+    using math::fvec2;
+    using math::fvec3;
+    using math::fvec4;
+    using math::f32vec2;
+    using math::f32vec3;
+    using math::f32vec4;
+    using math::dvec2;
+    using math::dvec3;
+    using math::dvec4;
+    using math::f64vec2;
+    using math::f64vec3;
+    using math::f64vec4;
+    using math::quat;
+    using math::asMatrix;
+    using math::fquat;
+    using math::dquat;
+    using math::translate;
+    using math::scale;
+    using math::rotate;
+    using math::transpose;
+    using math::fmat2x2;
+    using math::fmat3x3;
+    using math::fmat4x4;
+    using math::dmat2x2;
+    using math::dmat3x3;
+    using math::dmat4x4;
+    using math::decomposeTransformMatrix;
```

Comment:
The order of these is not fully identical to what is in the header. Perhaps it'd be nicer to split this up into types, aliases, and then functions. Or at least have some space between the "sections".

Reply:
Separated them by vector/quaternion/function, types/aliases/functions in order.